### ISSUE TITLE
Solid Color Layer design

### ISSUE THREAD

**Issue by don570** [ucupumar/ucupaint#186]
When I create a layer I normally choose Solid color  with an image mask.
However when I choose 'Solid Color'    I noticed the following --->

If a choose Roughness channel and then try to paint I get a confusing warning  about needing a texture (see image)
A texture can be an image,but it requires an expert to know that.  So I would like a better warning message.

![screenshot-ucupaint-solidcolor](https://github.com/user-attachments/assets/2c1da08f-f8d5-4ffa-bcbb-cd20ade614c3)

Second point-

I understand why it is important to allow a color channel with no associated image   i.e. less memory consumed.

I would suggest  allowing a layer that is solely a  'Solid Color' channel. Setting up a Roughness channel could be done but 
would be easier by selecting 'Solid Color with an image mask'
The dialog box can provide the instructions to add more channels 

The current way that Ucupaint has set up multiple channels works but it is confusing for a newbie.
Here is how I would design a Solid Color Layer dialog...
There would be only 3 things to choose  -->  name , Layer channel blending , and color
Maybe a fourth option - the level of opacity
Ucupaint's override feature would allow the overall roughness to be adjusted.
![screenshot-ucupaint-interface-fotor](https://github.com/user-attachments/assets/77491ff2-fc58-4ef4-a254-66af5ee0677f)

**Comment by don570** [comment]
I did more investigating of the Solid Color layer

I noticed that when I turn the Normal channel ON and OFF (with the check box)
that the bump of the layer underneath  turns OFF and ON

Normal channel ON    -->  Bump disappears
Normal channel OFF   --> Bump shows

I made a video to show
https://youtu.be/O5duJq7wMls

**Comment by ucupumar** [comment]
Ucupaint will have a UI redesign (#182), which will hopefully tackle multiple channels workflow confusion. Please follow the issue there, I'll open up for feedback when I already implemented some of the new UI features.

The solid color makes the bump values of the layer beneath it turned off because the default blending of the normal channel is `mix`, so the solid color will overwrite any bump value, you can try to change it to `add`. Solid color is mostly meant to be used with a mask though, except for some cases it can be useful for the bottom layer as a background.

**Comment by don570** [comment]
@ucupumar 
I stumbled across Blend Modes of the Normal channel and discovered something new.  I made a video that users will find useful. The layers underneath will show up but the solid color layer will still be seen as well when 'Compare Height' is chosen
![ucupaint-normal2](https://github.com/user-attachments/assets/ded28449-7f17-4b52-8675-1abfddf96f65)

Download:
https://youtu.be/NiCH8TRPwKc

**Comment by ucupumar** [comment]
Yeah `Compare Height` is very useful, it will make more sense if you enable the displacement setup in blender 4.2+, here's the demo about it https://www.youtube.com/watch?v=vC7jGM16t2Y
